launch/launch/action.py

```diff
@@ -24,8 +24,8 @@
 from .launch_description_entity import LaunchDescriptionEntity
 
 if False:
-    from .frontend import Entity
-    from .frontend import Parser
+    from .frontend import Entity  # noqa: F401
```

Comment:
I haven't seen this style of type annotations before. Is there a thread with more context? The annotations [are strings](https://github.com/ros2/launch/pull/274/files#diff-87194eeec55a75ef4357257623e817e5R52), but I would have expected them to be the class.

```
def parse(entity: Entity, parser: Parser):
```

 It's legal python since the [annotation can be any expression](https://www.python.org/dev/peps/pep-3107/#parameters), but I haven't seen any examples like this before.

Reply:
Nevermind, found an example. It's for avoiding import cycles

https://mypy.readthedocs.io/en/latest/common_issues.html#import-cycles